src/core/appdir.cpp

```diff
@@ -767,6 +769,10 @@ namespace linuxdeploy {
                 return d->deployFile(from, to, true);
             }
 
+            void AppDir::symlinkFile(const bf::path& target, const bf::path& symlink, const bool useRelativePath) {
```

Comment:
Can you please rename the function to e.g., `createSymlink`? That'd fit the name scheme better (`deploy*` register I/O operations for later execution, `create*` actually perform these tasks).
We should also rename the hidden function likewise.